lessonbuilder/hbm/src/java/org/sakaiproject/lessonbuildertool/SimplePageItemImpl.java

```diff
@@ -179,9 +179,7 @@ public SimplePageItemImpl(long pageId, int sequence, int type, String sakaiId, S
     // are too long. The main case this will happen is if we generate a name, which we do for URLs
     // and uploaded files.
 	private String maxlength(String s, int maxlen) {
-	    if (s == null)
-		s = "";  // oracle turns "" into null
-	    int len = s.length();
+	    int len = s==null?0:s.length();
```

Comment:
Stretch this out a bit? s == null ? 0 : s.length(). It's clearer, I think.

Reply:
Done